docs/semgrep-code/findings.md

```diff
@@ -25,12 +24,55 @@ Object.entries(frontMatter).filter(
 }
 </ul>
 
-# Manage findings in Semgrep Cloud Platform
+Semgrep Code generates a **finding** when a rule matches a piece of code in your codebase. You can use Semgrep Cloud Platform's [**Findings** page](https://semgrep.dev/orgs/-/findings) page to view all of the findings generated by Semgrep Code after it scans your codebase.
+
+![Semgrep Cloud Platform Findings page](/img/app-findings-overview.png#md-width)<br />
+
+## Navigate to the Findings page
+
+To access the [**Findings** page](https://semgrep.dev/orgs/-/findings):
+
+1. [Log into Semgrep Cloud Platform](https://semgrep.dev/login)
+2. Click **[Code](https://semgrep.dev/orgs/-/findings)** to navigate to the **Findings** page.
+
+## Findings page structure
+
+The **Findings** page consists of:
+
+- The filter panel, which you can use to group and filter for specific findings
+- Information about findings identified by Semgrep Code. Each  finding in the list includes:
+  - The name and description of the rule used to generate the finding
+  - The name of the project, as well as links to the Git branch and source code, where Semgrep Code identified the finding
 
-The Findings page enables you to easily manage findings. **Findings** are the core results of Semgrep analysis. Findings are generated when a Semgrep rule matches a piece of code.
+### Group findings
 
-![Semgrep Cloud Platform Findings page](/img/app-findings-overview.png)<br />
-*Figure 1.* Screenshot of findings page.
+By default, Semgrep groups all of the findings by the rule Semgrep used to match the code. The rules that reported the highest number of findings are at the top of the page. This helps you see which rules have the highest efficiency and which generate a lot of noise in your results.
+
+![Screenshot of the Findings page with findings grouped by rule](/img/app-findings.png#md-width)
+
+To view findings individually, toggle **Group by Rule** to **No grouping** using the drop-down menu in the header. Findings are displayed based on the date they were found, with the most recent finding listed at the top.
+
+![Screenshot of the Group by Rule option](/img/cloud-platform-findings-no-grouping.png#md-width)
+
+### Filter findings
+
+Semgrep offers multiple filters you can use to narrow down your results. The following criteria are available for filtering:
+
+| Filter                 | Description  |
+| ---------------------  | ------------ |
+| **Projects**           | Filter by codebases connected to Semgrep Cloud Platform. |
```

Comment:
```suggestion
| **Projects**           | Filter by repositories connected to Semgrep Cloud Platform. |
```
Anything that can be scanned and uploaded to Semgrep Cloud Platform (`semgrep ci`) is a git repo.